etna/blog/blocks.py

```diff
@@ -0,0 +1,67 @@
+from django.core.exceptions import ValidationError
+
+from wagtail import blocks
+
+from etna.core.blocks import (
+    ContactBlock,
+    ContentImageBlock,
+    ContentTableBlock,
+    DocumentsBlock,
+    ImageGalleryBlock,
+    ParagraphBlock,
+    QuoteBlock,
+    SectionDepthAwareStructBlock,
+    SubHeadingBlock,
+    SubSubHeadingBlock,
+    YouTubeBlock,
+)
+
+from ..media.blocks import MediaBlock
+
+
+class SectionContentBlock(blocks.StreamBlock):
+    contact = ContactBlock()
```

Comment:
Do blog posts need the potential for a contact block?

Reply:
Lizzy asked for the contact block to be on the blog posts https://national-archives.atlassian.net/browse/DSRC-121